layouts/company/list.html

```diff
@@ -17,15 +17,32 @@
              Normalize company name for sorting by uppercasing and translating accented characters to equivalent Latin
              alphabet characters.
         */}}
-        {{ $clean_name := replaceRE "[ÅÁÀÂÄÃ]" "A" (upper .Params.name) }}
-        {{ $clean_name := replaceRE "[Ç]" "C" $clean_name }}
-        {{ $clean_name := replaceRE "[ÉÈÊË]" "E" $clean_name }}
-        {{ $clean_name := replaceRE "[ÌÍÎÏ]" "I" $clean_name }}
-        {{ $clean_name := replaceRE "[ÒÓÔÕÖØ]" "O" $clean_name }}
-        {{ $clean_name := replaceRE "[ŬÙÚÛÜ]" "U" $clean_name }}
-        {{ $clean_name := replaceRE "[Ñ]" "N" $clean_name }}
+        {{ $clean_name := replaceRE "[ÅÁÀÂÄÃÅÆΆ]" "A" (upper .Params.name) }}
+        {{ $clean_name := replaceRE "[B̌B́]" "B" $clean_name }}
+        {{ $clean_name := replaceRE "[ÇČĆ̣Ḉ]" "C" $clean_name }}
+        {{ $clean_name := replaceRE "[ĎÐD́]" "D" $clean_name }}
+        {{ $clean_name := replaceRE "[ÉÈÊËĚÊ̌ΕĖ́Ę́É̩Έ]" "E" $clean_name }}
```

Comment:
```suggestion
        {{ $clean_name := replaceRE "[ÉÈÊËĚÊ̌ΕĖ́Ę́É̩]" "E" $clean_name }}
```
The capital epsilon was duplicated. Or did I miss sth?

Reply:
yeah last one is not epsilon, it's Έ

Reply:
oops, my bad. To be precise, seems like its both: https://www.fileformat.info/info/unicode/char/0388/index.htm But we want to keep it. :)